web/src/test/java/org/cbioportal/weblegacy/CNSegmentControllerConfig.java

```diff
@@ -47,12 +47,7 @@
 @EnableWebMvc
 @ComponentScan(basePackages = {"org.cbioportal.weblegacy"}, resourcePattern = "**/*CNSegmentController.class")
 public class CNSegmentControllerConfig extends WebMvcConfigurerAdapter {
-    @Override
-    public void configureMessageConverters(List<HttpMessageConverter<?>> converters) {
-        MappingJackson2HttpMessageConverter mappingJackson2HttpMessageConverter = new MappingJackson2HttpMessageConverter();
-        mappingJackson2HttpMessageConverter.setObjectMapper(new CustomObjectMapper());
```

Comment:
Remove attempt to link to a CustomObjectMapper ... not needed? And what about the connected mixins? In this case there is a mixin defined for CopyNumberSegments configured through the web package. Perhaps this is not needed for test code?